pb/api/v1/data.proto

```diff
@@ -164,3 +158,38 @@ message Resource {
   string name = 1;
   string link = 2;
 }
+
+message ListTabSummariesRequest {
+  // Scope defines the GCS bucket to read the results from.
+  string scope = 1;
+  // Name of the dashboard to fetch tab summaries for.
+  string dashboard = 2;
+}
+
+message ListTabSummariesResponse {
+	// List of tab summaries.
+	repeated TabSummary tab_summaries = 1;
+}
+
+message TabSummary {
+	// The name of the dashboard this tab belongs to.
```

Comment:
nit: Not aligned with other fields?

Reply:
That's funny, since it was aligned properly in my branch :) I will try to push some changes and let's see if this gets sorted out.

Reply:
Still looks borked on GitHub. Make sure you (more likely, your text editor) aren't mixing tabs and spaces.